conans/client/toolchain/msbuild.py

```diff
@@ -2,10 +2,44 @@
 import textwrap
 from xml.dom import minidom
 
+from conans.client.toolchain.visual import vcvars_arch, vcvars_command
+from conans.client.tools import msvs_toolset
 from conans.errors import ConanException
 from conans.util.files import save, load
 
 
+class MSBuildCmd(object):
```

Comment:
Are we placing these "new build helpers" in the toolchain folder as well?

Reply:
It is just a way (for me, at the moment) to not mix them with the old ones, and we are sure everything inside the ``toolchain`` folder is experimental.